website/content/en/status/report-2023-01-2023-03/daemon.adoc

```diff
@@ -0,0 +1,27 @@
+=== daemon(8) improvements
+
+Links: +
+man:daemon[8] URL: link:https://man.freebsd.org/cgi/man.cgi?query=daemon&sektion=8&format=html[https://man.freebsd.org/cgi/man.cgi?query=daemon&sektion=8&format=html
```

Comment:
The first link is not closing correctly and mixing with the second one. Since the name of the link is the same as the displayed text, there's no need to repeat it. While we're at it, we could also consider using a shorter URL for the man pages to make it look better. 

```
Links: +
man:daemon[8] URL: link:https://man.freebsd.org/daemon/8[] +
link:https://libera.chat/[Libera IRC] URL: link:https://libera.chat/[]
```

Other than that, everything looks good. We can fix the issue with the links when committing the changes.

Thanks for your work.

Reply:
fixed